SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java

```diff
@@ -763,6 +763,8 @@ protected void loadCampaignsFromJSON(JSONObject json, JSONObject settings) {
 
             Map<Integer, String> campaignsDownloaded = null;
 
+            boolean wasQAUser = (qaUser != null);
```

Comment:
I prefer something like wasPreviouslyAQAUser.
Its not a nice name, so as Shane says, best to add a comment.